content/com/github/ldapchai/ldapchai-0.8.6.jar.diffoscope

```diff
@@ -0,0 +1,1068 @@
+│ │ -  #35 = Utf8               ()[B
+│ │ -  #36 = Utf8               SourceFile
+│ │ -  #37 = Utf8               ApacheLdapProviderImpl.java
+│ │ -  #38 = Utf8               EnclosingMethod
+│ │ -  #39 = Class              #40            // com/novell/ldapchai/provider/ApacheLdapProviderImpl
+│ │ -  #40 = Utf8               com/novell/ldapchai/provider/ApacheLdapProviderImpl
+│ │ -  #41 = NameAndType        #42:#43        // extendedOperation:(Ljavax/naming/ldap/ExtendedRequest;)Ljavax/naming/ldap/ExtendedResponse;
+│ │ -  #42 = Utf8               extendedOperation
+│ │ -  #43 = Utf8               (Ljavax/naming/ldap/ExtendedRequest;)Ljavax/naming/ldap/ExtendedResponse;
+│ │ -  #44 = Utf8               InnerClasses
+│ │ +  #31 = Utf8               Signature
+│ │ +  #32 = Utf8               getID
+│ │ +  #33 = Utf8               getEncodedValue
+│ │ +  #34 = Utf8               ()[B
+│ │ +  #35 = Utf8               SourceFile
+│ │ +  #36 = Utf8               ApacheLdapProviderImpl.java
+│ │ +  #37 = Utf8               EnclosingMethod
+│ │ +  #38 = Class              #39            // com/novell/ldapchai/provider/ApacheLdapProviderImpl
+│ │ +  #39 = Utf8               com/novell/ldapchai/provider/ApacheLdapProviderImpl
+│ │ +  #40 = NameAndType        #41:#42        // extendedOperation:(Ljavax/naming/ldap/ExtendedRequest;)Ljavax/naming/ldap/ExtendedResponse;
+│ │ +  #41 = Utf8               extendedOperation
+│ │ +  #42 = Utf8               (Ljavax/naming/ldap/ExtendedRequest;)Ljavax/naming/ldap/ExtendedResponse;
+│ │ +  #43 = Utf8               InnerClasses
+│ │  {
+│ │    final org.apache.directory.api.ldap.model.message.ExtendedResponse val$apacheResponse;
+│ │      descriptor: Lorg/apache/directory/api/ldap/model/message/ExtendedResponse;
+│ │      flags: (0x1010) ACC_FINAL, ACC_SYNTHETIC
+│ │  
+│ │    final com.novell.ldapchai.provider.ApacheLdapProviderImpl this$0;
+│ │      descriptor: Lcom/novell/ldapchai/provider/ApacheLdapProviderImpl;
+│ │ @@ -63,41 +62,37 @@
+│ │  
+│ │    com.novell.ldapchai.provider.ApacheLdapProviderImpl$2();
+│ │      descriptor: (Lcom/novell/ldapchai/provider/ApacheLdapProviderImpl;Lorg/apache/directory/api/ldap/model/message/ExtendedResponse;)V
+│ │      flags: (0x0000)
+│ │      Code:
+│ │        stack=2, locals=3, args_size=3
+│ │           0: aload_0
+│ │ -         1: aload_2
+│ │ -         2: putfield      #1                  // Field val$apacheResponse:Lorg/apache/directory/api/ldap/model/message/ExtendedResponse;
+│ │ +         1: aload_1
+│ │ +         2: putfield      #1                  // Field this$0:Lcom/novell/ldapchai/provider/ApacheLdapProviderImpl;
+│ │           5: aload_0
+│ │ -         6: aload_1
+│ │ -         7: putfield      #7                  // Field this$0:Lcom/novell/ldapchai/provider/ApacheLdapProviderImpl;
+│ │ +         6: aload_2
+│ │ +         7: putfield      #7                  // Field val$apacheResponse:Lorg/apache/directory/api/ldap/model/message/ExtendedResponse;
```

Comment:
The statements in the constructor are swapped. Although they are independent of each other, they are still semantically non-equivalent.

```diff
-    ApacheLdapProviderImpl$2(final ApacheLdapProviderImpl this$0, final org.apache.directory.api.ldap.model.message.ExtendedResponse var2) {
+    ApacheLdapProviderImpl$2(ApacheLdapProviderImpl this$0, org.apache.directory.api.ldap.model.message.ExtendedResponse var2) {
-     this.val$apacheResponse = var2;     
      this.this$0 = this$0;
+    this.val$apacheResponse = var2;
   }
```